reporting/impl/src/main/java/org/jboss/windup/reporting/export/ExportCSVFileRuleProvider.java

```diff
@@ -202,5 +171,34 @@ private String buildLinkString(Iterable<LinkModel> links)
             linksString.toString();
             return linksString.toString();
         }
+
+        private void writeCsvRecordForProject(String outputFolderPath, ProjectModel projectModel, String[] line)
+        {
+            if (!projectToFile.containsKey(projectModel.getName()))
+            {
+                CSVWriter writer = initCSVWriter(outputFolderPath + PathUtil.cleanFileName(projectModel.getName()) + ".csv");
+                projectToFile.put(projectModel.getName(), writer);
+            }
+            projectToFile.get(projectModel.getName()).writeNext(line);
+
+        }
+
+        private CSVWriter initCSVWriter(String path)
+        {
+            try
+            {
+                CSVWriter writer = new CSVWriter(
+                            new FileWriter(path), ';');
+                String[] headerLine = new String[] { "Rule Id", "Problem type", "Title", "Description", "Links", "Application", "File Name",
+                            "File Path", "Line", "Story points" };
+                writer.writeNext(headerLine);
+                return writer;
+            }
+            catch (IOException e)
+            {
+                System.err.println("Windup was not able to create a CSV file " + path + ". CSV Export will not be generated.");
+                throw new WindupException("Unable to create file " + path, e);
```

Comment:
I don't know. I am a little worried that when people specify --exportCSV and will not be given CSV, they will think windup is broken